samples/cellular/modem_shell/src/gnss/gnss.c

```diff
@@ -667,6 +675,7 @@ static void get_agps_data(struct k_work *item)
 		.rx_buf = rx_buf,
 		.rx_buf_len = sizeof(rx_buf),
 	};
+#endif
```

Comment:
Guard for COAP could be explicit here

Reply:
Hmm, I commented on an endif here. That seems like a mistake on my part. Possibly I grabbed the wrong line, one sec.

Reply:
IDK why I thought this was a coap branch, it clearly isn't. My bad.